go/vt/sqlparser/ast.go

```diff
@@ -136,8 +144,8 @@ type Insert struct {
 }
 
 func (node *Insert) Format(buf *TrackedBuffer) {
-	buf.Myprintf("insert %vinto %v%v %v%v",
-		node.Comments,
+	buf.Myprintf("insert %v%sinto %v%v %v%v",
```

Comment:
We don't. AST_IGNORE already contains the space. Otherwise, this code would become conditional because the space would be needed only if IGNORE was not empty.